CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - (#267) Aliases like `git amend` are now installed only if the user does not already have aliases with the same name. Thanks to @rslabbert for implementing this.
 - Improved performance up to 15x for `git restack` on large commit histories.
+- (#280) Use git2's get_short_oid function to avoide commit hash collision.
```

Comment:
```suggestion
- (#280) Use `git2`'s `short_id` function to disambiguate commit hashes when printed.
```

Reply:
What is the changelog's target audience? If it's end users, then it probably doesn't make sense to mention `git2` here. Say something about always making them unambiguous and to respect `core.abbrev` (if I understood correctly what this PR means to the end user).

Reply:
@martinvonz good point; I've updated the changelog with a more user-facing description. (Oops, I forgot to mention `core.abbrev`, but it's probably not too important.)